### ISSUE TITLE
Is it possible to search only the question?

### ISSUE THREAD

**Issue by imthebat** [jontyankit/TriviaHack#1]

**Comment by jontyankit** [comment]
Yes its possible to search only for question. For that you have to make changes in https://github.com/jontyankit/TriviaHack/blob/master/app/src/main/java/com/ankit/game/hack/AccessibilityDispatcher.java
Don't add answer String in StringBuilder.
